kitsune/sumo/utils.py

```diff
@@ -328,3 +329,16 @@ def is_ratelimited(request, name, rate, method=['POST'], skip_if=lambda r: False
             Record.objects.info('sumo.ratelimit', '{key} hit the rate limit for {name}',
                                 key=key, name=name)
     return request.limited
+
+
+def get_browser(request):
+    user_agent = request.META.get('HTTP_USER_AGENT', '')
+    user_agent = str(user_agent)
+
+    try:
+        browser = [[b.group(1), b.group(2)] for b in re.finditer(
+            r'(?i)(firefox|msie|chrome|safari)[\/\s]([\d.]+)', user_agent)][0]
+        browser = browser[0]
```

Comment:
Instead of using `.finditer` and catching the index error here, you should do something like

```python
match = re.find(r'...', user_agent)
if match:
    browser = match.group(1)
else:
    browser = None
```